squizlabs/php_codesniffer/3.6/manifest.json

```diff
@@ -0,0 +1 @@
+../3.0/manifest.json
```

Comment:
Let's copy this file instead. It will be easier to read and maintain.

Reply:
Done.